### ISSUE TITLE
DC screws

### ISSUE THREAD

**Issue by firolightfog** [docb/dbRackModules#32]
<img width="265" alt="image" src="https://github.com/docb/dbRackModules/assets/34127628/90b61e3a-ff6b-4970-b009-c9551d758147">

OK, I know it's not the most important thing in the world but I think you can drop those screws. :)

**Comment by docb** [comment]
yes, i will remove all screws from my modules if i get the time!
